src/ONYXKEYS.ts

```diff
@@ -163,6 +163,9 @@ const ONYXKEYS = {
     /** This NVP contains the referral banners the user dismissed */
     NVP_DISMISSED_REFERRAL_BANNERS: 'nvp_dismissedReferralBanners',
 
+    /** This NVP contains the user intent to not show the instant submit explanation modal again */
```

Comment:
Note: This NVP ALSO stores "has the user seen the modal before"
- `undefined` = the user has never seen it before (it can't be dismissed yet)
- `false` = the user has seen it before, but the user hasn't chosen "Don't show me this again"
- `true` = the user has seen it before & the user choose "Don't show me this again"

Reply:
Made some edits to the suggestion above, hope you don't mind :D

Reply:
It looks like these changes weren't applied?